CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.3 (2019-04-25)
+
+- Fixed flyout component to not close when clicking on internal components. [#30](https://github.com/blackbaud/skyux-flyout/pull/30)
```

Comment:
My bad. Went to the tab I had open.

Fixed flyout component to remain open when users click internal components.

Reply:
That looks good to me.